app/experimenter/nimbus-ui/src/components/AppLayoutWithSidebar/index.tsx

```diff
@@ -31,45 +46,94 @@ export const AppLayoutWithSidebar = ({
           className="bg-light pt-2 border-right shadow-sm"
         >
           <nav data-testid="nav-sidebar">
-            <Nav className="flex-column" as="ul">
-              <LinkNav storiesOf="pages/Home" className="mb-2">
+            <Nav className="flex-column font-weight-semibold" as="ul">
+              <LinkNav
+                storiesOf="pages/Home"
+                className="mb-3 small font-weight-bold"
+                textColor="text-secondary"
+              >
+                <ChevronLeft className="ml-n1" width="18" height="18" />
                 Experiments
               </LinkNav>
               <LinkNav
                 route={`${slug}/edit/overview`}
                 storiesOf="pages/EditOverview"
                 testid="nav-edit-overview"
               >
+                <Cog className="mr-3" width="18" height="18" />
                 Overview
+                {review?.missingFields["Overview"] && (
+                  <AlertCircle
+                    className="ml-3"
+                    width="18"
+                    height="18"
+                    data-testid="missing-detail-alert-overview"
+                  />
+                )}
               </LinkNav>
               <LinkNav
                 route={`${slug}/edit/branches`}
                 storiesOf="pages/EditBranches"
                 testid="nav-edit-branches"
               >
+                <Layers className="mr-3" width="18" height="18" />
                 Branches
+                {review?.missingFields["Branches"] && (
+                  <AlertCircle
+                    className="ml-3"
+                    width="18"
+                    height="18"
```

Comment:
Since these 3 props are repeated so much, you _could_ add something like this above `AppLayoutWithSidebar`:

```
const AlertCircleIcon = ({ testId }: { testId: string }) => (
  <AlertCircle
    className="ml-3"
    width="18"
    height="18"
    data-testid={`missing-detail-alert-${testId}`}
  />
);
```

Then you'd just `<AlertCircleIcon testId="overview" />`.

You could maybe DRY up the className/width/height of all other icons too, but not necessary.

Reply:
recent change removed the need for this